_docs/response-templating.md

```diff
@@ -631,6 +631,19 @@ Likewise decimals can be produced with or without bounds:
 
 {% endraw %}
 
+## Fake data helpers
+
+These helpers produce random fake data of the desired types available in the [Data Faker library]([url](https://github.com/datafaker-net/datafaker)).
```

Comment:
It would be nice to document how to get this extension, because it is not an out of the box one

Reply:
Yes, I had written this when I had this implemented this feature as a core helper instead of an extension. Have marked this PR as draft and will update as per comments, thanks!

Reply:
updated!